deployer/src/main.rs

```diff
@@ -72,6 +76,10 @@ async fn main() {
         Backend::Deployer,
     );
 
+    let resource_manager = ResourceManager::new(&args.provisioner_address)
+        .await
+        .unwrap();
```

Comment:
I think we can remove this ResourceManager struct, and rather pass the provisioner address to persistence, like we do with resource recorder at the start of this function (main). Then we put the provisioner client in the persistence struct, like we do for the resource manager client.

Reply:
Looking into this.

Reply:
This should be covered in the latest changes.